Source/Core/Core/IPC_HLE/WII_Socket.cpp

```diff
@@ -6,6 +6,7 @@
 #ifndef _WIN32
 #include <unistd.h>
 #endif
+#include <arpa/inet.h>
```

Comment:
This shouldn't be included on non-POSIX systems. It makes the Windows builds fail.